### ISSUE TITLE
@vector-copy! routines are not in manual.

### ISSUE THREAD

**Issue by gambiteer** [gambit/gambit#559]
I don't have time right now to figure out how to add then, sorry.

**Comment by lassik** [comment]
Are these the [SRFI 160](https://srfi.schemers.org/srfi-160/srfi-160.html) routines?

**Comment by lassik** [comment]
The manual is in the [Texinfo](https://www.gnu.org/software/texinfo/) file `doc/gambit.txi`. Would be nice if we got up to speed on giving it an overhaul :)

See also #539

**Comment by feeley** [comment]
Definitely! In my lab I've started brainstorming on the possibility of moving the documentation to another, more web-friendly, format.  We're looking at various other sites for inspiration (racket docs, python docs, mozilla docs).  Given that the JS backend is in relatively good shape, I'd like the code examples to be runnable directly from the documentation.  Also it would be nice to have doc strings in the runtime library to automatically generate the documentation.  So we need a markup language... and given the close ties with github I was thinking of using that.

**Comment by daviwil** [comment]
Markdown is a great option for simple markup, plus there are many static website generators you can use to produce a nice website from the output. It'd be pretty easy to inject runnable samples in that type of tool chain.

**Comment by feeley** [comment]
Can you point me to some examples?

**Comment by lassik** [comment]
Markdown has the allure of being widely used in Wikis, issues and readmes, but I would like to caution against it since it's heavily optimized towards easy uptake at the expense advanced tasks. It also has split into different dialects for advanced features like tables.

I aspire to eventually serve all the major Scheme implementations' manuals from `doc.scheme.org` (and programmatically from `api.scheme.org`) with a full browser and index. This involves parsing all the manuals to extract things like intra-document links, table of contents, section boundaries, etc. To that end, I did a survey last year of what documentation formats Schemes are using: http://practical-scheme.net/wiliki/schemexref.cgi?Concept%3ADocumentationFormat Texinfo is the clear winner there.

Since Texinfo is popular in the Scheme community and also designed with a view to complex documentation (while still being more approachable than TeX) and machine extraction, I would recommend staying with it even if it's not as easy as Markdown. The extra features and design attention to detail will help down the road.

If Texinfo is too hard to pick up, it could be productive to write a Texinfo primer and "cheat sheet" if there aren't good ones already. To me it seems like one of the most approachable markup languages I've seen.

It would also help significantly if a Texinfo rendered could be added to GitHub's user interface. GitHub has a plug-in architecture which currently renders Markdown, AsciiDoc and probably other formats. @daviwil if you happend to know who we could contact at GitHub to have a Texinfo renderer added and what it would entail, that would be wonderful.

**Comment by lassik** [comment]
Ah, here it is: https://github.com/github/markup
